src/command_modules/azure-cli-configure/azure/cli/command_modules/configure/custom.py

```diff
@@ -6,15 +6,15 @@
 from __future__ import print_function
 import os
 import sys
-from six.moves import input, configparser #pylint: disable=redefined-builtin
+from six.moves import configparser #pylint: disable=redefined-builtin
```

Comment:
PEP8: two spaces between the '#' and the previous char, one space between the '#' and the following char.